### PR TITLE
[MM-60829] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/unarchive_channel_modal/unarchive_channel_modal.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by KvngMikey**
#### Summary
The goal of this PR is to remove all instances of “FormattedMarkdownMessage” (imported from “components/formatted_markdown_message”) in the file webapp/channels/src/components/unarchive_channel_modal/unarchive_channel_modal.tsx. 

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28497
Jira https://mattermost.atlassian.net/browse/MM-60829

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Please run i18n-extract for changes in en.json also there are few CI failures as well

**Review by JulienTant — APPROVED** [review]
Thanks @KvngMikey

File: webapp/channels/src/components/unarchive_channel_modal/unarchive_channel_modal.tsx [hunk 1]
```diff
@@ -70,11 +68,12 @@ export default class UnarchiveChannelModal extends React.PureComponent<Props, St
                 </Modal.Header>
                 <Modal.Body>
                     <div className='alert alert-danger'>
-                        <FormattedMarkdownMessage
-                            id='unarchive_channel.viewArchived.question'
-                            defaultMessage={'Are you sure you wish to unarchive the **{display_name}** channel?'}
+                        <FormattedMessage
+                            id='unarchive_channel_viewArchived.question'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/unarchive_channel_modal/unarchive_channel_modal.tsx:72]:
```suggestion
                            id='unarchiveChannelModal.viewArchived.question'
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/unarchive_channel_modal/unarchive_channel_modal.tsx (hunk 1 above)
Comment (KvngMikey) [reply webapp/channels/src/components/unarchive_channel_modal/unarchive_channel_modal.tsx]: done, updated @M-ZubairAhmed

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by yasserfaraazkhan — APPROVED** [review]
@M-ZubairAhmed 
can you check the e2e tests? might be unrelated to this PR but most failed tests have same error. Just want to make sure its not related to PR

image

**Comment by M-ZubairAhmed** [comment]
> @M-ZubairAhmed can you check the e2e tests? might be unrelated to this PR but most failed tests have same error. Just want to make sure its not related to PR


Cross checked, yes thats unrelated to this PR, since this is from unarchive modal and the test failed on channel search

**Pull request merged**